dashboard/modules/job/job_log_storage_client.py

```diff
@@ -13,45 +17,65 @@ class JobLogStorageClient:
     """
 
     # Number of last N lines to put in job message upon failure.
-    NUM_LOG_LINES_ON_ERROR = 10
-    # Maximum number of characters to print out of the logs to avoid
+    MAX_LOG_LINES_ON_ERROR = 10
+    # Max number of characters to print out of the logs to avoid
```

Comment:
let's please avoid pure renaming/refactoring in addition to logic changes. it makes our lives harder as reviewers to cut through the noise and increases the chance of bugs slipping through